controllers/ajax.py

```diff
@@ -1726,6 +1727,24 @@ def get_question_source():
             else:
                 questionid = random.choice(list(possible))
 
+    logger.debug(f"toggle is {toggle}")
+    if toggle:
+        prev_selection = (
+            db(
+                (db.selected_questions.sid == auth.user.username)
+                & (db.selected_questions.selector_id == selector_id)
+            )
+            .select()
+            .first()
+        )
+        if prev_selection:
+            questionid = prev_selection.selected_id
+        else:
+            questionid = request.vars["questions"].split(",")[0]
+    else:
```

Comment:
This else is wrong.  the `if toggle` is at the outermost level of the function and this is keeping the rest of the function from executing

Reply:
It seems like there must have been some other error checking going on?  I just commented out the else and all of our current unit tests for get_question_source do pass.